shows/481 - Supperclub Voice coding.md

```diff
@@ -41,10 +41,10 @@ Directus is an open-source data platform that instantly layers on top of any SQL
 * ["Incremental, zero-config Code Nav using stack graphs" by Douglas Creager](https://www.youtube.com/watch?v=l2R1PTGcwrE)
 * **[35:20](#t=35:20)** Does eye tracking work?
 * **[36:48](#t=36:48)** What kind of mic do you use?
-* [DPA Microphone](https://www.bhphotovideo.com/c/product/1407826-REG/dpa_microphones_4188_dc_f_f00_lh_d_fine_core_4188_slim.html)
+* [DPA Microphone](https://www.sweetwater.com/store/detail/4288HSBg--dpa-dfine-core-4288-omnidirectional-flex-headset-microphone-with-microdot-connector-long-length-b)
```

Comment:
This is the exact model number of my mic, and people in the Talon slack tend to recommend Sweetwater as a seller for these mics